src/main/events/library/open-game.ts

```diff
@@ -21,8 +24,31 @@ const openGame = async (
 
   if (fs.existsSync(gamePath)) {
     const setupPath = path.join(gamePath, "setup.exe");
+    console.log(setupPath);
     if (fs.existsSync(setupPath)) {
-      shell.openExternal(setupPath);
+      if (process.platform === "win32") {
+        shell.openExternal(setupPath);
+      } 
+      if (process.platform === "linux") {
+        if (spawnSync('which', ['lutris']).status === 0) {
```

Comment:
Please use `exec` for short-living processes instead of `spawn`:
https://stackoverflow.com/questions/48698234/node-js-spawn-vs-execute